extensions/amp-font/0.1/fontloader.js

```diff
@@ -217,7 +217,7 @@ export class FontLoader {
     const customFontElement = this.customFontElement_ =
         this.document_.createElement('div');
     style.setStyles(customFontElement, {
-      fontFamily: this.fontConfig_.family,
+      fontFamily: this.fontConfig_.family + ' ,san-serif, serif',
```

Comment:
use

`this.fontConfig_.family + ',' + DEFAULT_FONTS_.join(',')`

https://github.com/zhouyx/amphtml/blob/0ef32eaf6b3a23fcfc826d2516b42ec82d0d3efd/extensions/amp-font/0.1/fontloader.js#L31